sbncode/EventGenerator/MeVPrtl/config/numi_kaon_common.fcl

```diff
@@ -25,6 +25,28 @@ numi_kaon: {
   RandomizeFiles: true
 }
 
+numi_tau: {
+  tool_type: TxtFileGen
+  SearchPath: "/pnfs/icarus/persistent/users/gputnam/numi_unconventional_fluxes/"
```

Comment:
Can we have examples of fluxes in the PR itself? I don't like that the PR points to a personal user space